.gitlab/ci.sh

```diff
@@ -31,29 +31,29 @@ mkdir -p "$CABAL_DIR"
 #
 # $PLATFORM comes from CI.
 if [ "$(getconf LONG_BIT)" = "32" -o "${PLATFORM:=xxx}" = "x86_64-linux-centos7" ] ; then
-    echo 'constraints: lukko -ofd-locking' >> cabal.project.release.local
+    echo 'constraints: lukko -ofd-locking' >> cabal.release.project.local
 fi
 
 # In February 2024, cabal started using zlib-0.7.0.0, which uses pkg-config by
 # default. The GitLab CI environment doesn't (yet) supply pkg-config, and zlib
 # does just fine without it on modern GHCs. That said, the CI environment
 # probably *should* have pkg-config installed. See
 # https://github.com/haskell/cabal/issues/9774.
-echo 'constraints: zlib -pkg-config' >> cabal.project.release.local
+echo 'constraints: zlib -pkg-config' >> cabal.release.project.local
 # Furthermore, on Windows, zlib claims that libz is shipped with GHC, so it just
 # uses @extra-libraries: z@ if pkg-config is False. If you are reading this
 # comment, however, this didn't work. Thus we switch to using the bundled libz,
```

Comment:
In this file, why `.project.local` and not `.local.project`?

Reply:
Isn't that normal for local overrides to a project file (and enforced by cabal)?